### PR TITLE
Improve main README about the conf.yaml

### PR DESCRIPTION
**Pull request by mpagot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,16 +24,20 @@ pip install -r requirements.txt
...
 ssh-add id_rsa_cloud
 ```
 
 ## Usage
 
-### Build and Destroy Terraform and Ansible components
+### Qesap driven deployment
 
-To get started, the user must create a yml configuration file
+This project provide a script in `scripts/qesap/qesap.py` to drive the deployment, attempting to hide the complexity f the underling Terraform and Ansible scripts and commands.
```
Comment (alvarocarvajald) [inline README.md:36]: s/provide/provides/
s/complexity f the underling/complexity of the underlying/

File: README.md [hunk 2]
```diff
@@ -47,6 +51,97 @@ Run the `config` step to get all the needed Terraform and Ansible configuration
...
+result in a `terraform/azure/terraform.tfvars` generated file like:
+
+```bash
+deploy_name = "something"
+region = "europe"
+```
+
+##### Ansible settings
+
+The Ansible playbooks needs some .yaml configuration files. Some of them are geenrated by Terraform, some of them has to be provided by the user. The **qesap.py** `configure` command can support the user to create them.
```
Comment (alvarocarvajald) [inline README.md:83]: s/geenrated/generated/

File: README.md [hunk 3]
```diff
@@ -47,6 +51,97 @@ Run the `config` step to get all the needed Terraform and Ansible configuration
...
+
+###### Hana variables
+
+The Ansible project require a configuration file `ansible/playbooks/vars/hana_vars.yaml`. It is generated from the `ansible::hana_vars` section of the config.yaml:
```
Comment (alvarocarvajald) [inline README.md:87]: s/require/requires/

File: README.md [hunk 4]
```diff
@@ -55,6 +150,14 @@ Terraform and Ansible deployment steps can be executed like:
 (venv) python3 scripts/qesap/qesap.py --verbose -c config.yaml -b <FOLDER_OF_YOUR_CLONED_REPO> ansible
 ```
 
+the terraform sub command has a partial support for Terraform workspace
```
Comment (alvarocarvajald) [inline README.md:153]: s/the/The/

File: README.md [hunk 5]
```diff
@@ -63,9 +166,12 @@ The destruction of the infrastructure, including the de-registration of SLES, ca
...
 ```
 
-### Manual terraform deployment
+### Manual deployment
+
+It is possible to use the deployment, without to use the `qesap.py` script.
+A possible way to get the proper sequence of terraform and ansible commands to run, can be obtained from `qesap.py` using the `--dryrun` mode.
 
-Here an example of Azure deployment
+Here an example of a sequence of Terraform commands to obtain the Azure deployment
```
Comment (alvarocarvajald) [inline README.md:174]: s/Here an/Here is an/

**Review by alvarocarvajald — APPROVED** [review]
LGTM

**Comment by jankohoutek** [comment]
It would be fine to have marked which is **required** a and which just **optional**

**Review by BillAnastasiadis — APPROVED** [review]

**Pull request merged**